actionview/test/template/tag_helper_test.rb

```diff
@@ -339,15 +339,21 @@ def test_tag_builder_with_unescaped_conditional_hash_classes
   end
 
   def test_class_names
-    assert_equal "song play", class_names(["song", { "play": true }])
-    assert_equal "song", class_names({ "song": true, "play": false })
-    assert_equal "song", class_names([{ "song": true }, { "play": false }])
-    assert_equal "song", class_names({ song: true, play: false })
-    assert_equal "song", class_names([{ song: true }, nil, false])
-    assert_equal "song", class_names(["song", { foo: false }])
-    assert_equal "song play", class_names({ "song": true, "play": true })
-    assert_equal "", class_names({ "song": false, "play": false })
-    assert_equal "123", class_names(nil, "", false, 123, { "song": false, "play": false })
+    [:class_names, :token_list].each do |helper_method|
+      helper = ->(*arguments) { public_send(helper_method, *arguments) }
+
+      assert_equal "song play", helper.(["song", { "play": true }])
+      assert_equal "song", helper.({ "song": true, "play": false })
```

Comment:
@p8 does this line cover the behavior you're describing in https://github.com/rails/rails/pull/40146#issuecomment-684782080?

Reply:
@seanpdoyle  Yes, it is. I can see this being useful for `class_names` or values for aria attributes.
I'm just not sure it is expected behaviour for a method called `token_list`.
But looking at `DOMTokenList` maybe it is.

Reply:
@p8 I'm thinking of something like this:

```html+erb
<%= form_with(model: post) do |form| %>
  <%= form.text_field :title, aria: { 
    describedby: token_list(:always_present_label, post_title_error: post.errors[:title].any?) }
  } %>
<% end %>
```

Reply:
That does look nice! 👍